x/ccv/types/ccv.go

```diff
@@ -101,5 +103,38 @@ func (cp ConsumerPacketData) ValidateBasic() (err error) {
 
 func (cp ConsumerPacketData) GetBytes() []byte {
 	bytes := ModuleCdc.MustMarshalJSON(&cp)
+
```

Comment:
This shim only works when the provider is on 45, we could also add an appropriate shim on the provider on it's 47 version